modules/drivers/oracle/src/metabase/driver/oracle.clj

```diff
@@ -301,6 +301,10 @@
       "XDB"
       "XS$NULL"}))
 
+(defmethod driver/metadata-escape-entity-name :oracle
```

Comment:
We usually go for the more narrative names like `escape-entity-name-for-metadata` or `-for-sync` rather than prefixing (took me a while to get comfortable with this style -- and sometimes I still lapse -- but am always thankful for it when reading code I'm not intimate with).